src/index.html

```diff
@@ -6,7 +6,5 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
   </head>
-  <body>
-    <div id="app" />
-  </body>
+  <body></body>
```

Comment:
It's because expecting "app" in an index used by multiple entries is a not super isomorphic, I've had good use just creating the dom element and putting it in the body instead.